src/qmckl_context.org

```diff
@@ -97,11 +97,11 @@ qmckl_context qmckl_context_copy(const qmckl_context context) {
    implying that the context has been freed.
 
    #+BEGIN_SRC C :tangle qmckl_context.h
-qmckl_context qmckl_context_destroy(qmckl_context context);
+qmckl_context qmckl_context_destroy(const qmckl_context context);
```

Comment:
Can you delete a const pointer ? I think delete is going to make it NULL.

Reply:
Yes, I was not so sure about this PR. I'll close it until I'm sure what I'm doing.